### ISSUE TITLE
Can't import the named export 'createElement'

### ISSUE THREAD

**Issue by mikeaustin** [developit/htm#103]
It's probably something I'm missing, but the example for react code is erroring with:

> ./node_modules/htm/react/index.mjs
> Can't import the named export 'createElement' from non EcmaScript module (only default export is available)

I've added the module via npm install, and it looks very basic. Why is it using index.mjs if I'm not using it as a javascript module? If I remove "module" and let it use "main": "index.js", it sill complains about import not being at the top.

```
import ReactDOM from 'react-dom'; 
import { html } from 'htm/react';
ReactDOM.render(html`<a href="/">Hello!</a>`, document.body);
```

**Comment by developit** [comment]
Hi @mikeaustin!

We need to re-release the current build in order to fix this, since we'll have to remove the `.mjs` extension. `.mjs` puts webpack into a strict mode, which is causing the issue you've run into here. 

One workaround would be to change your import to `'htm/react/index.js'`, or to use https://github.com/lukeed/webpack-modules.

**Comment by jviide** [comment]
@mikeaustin Now https://github.com/developit/htm/pull/110 from @developit (that fixes this) has now merged and released as a part of HTM 2.2 👍
